plugins/source/aws/docs/tables/aws_ram_resource_share_invitations.md

```diff
@@ -0,0 +1,25 @@
+# Table: aws_ram_resource_share_invitations
+
+https://docs.aws.amazon.com/ram/latest/APIReference/API_ResourceShareInvitation.html
+
+The primary key for this table is **resource_share_invitation_arn**.
```

Comment:
Could we rename the `resource_share_invitation_arn` column to `arn` please, to be consistent with other tables :bow: